bcda/models/models.go

```diff
@@ -96,138 +72,6 @@ func (job *Job) CheckCompletedAndCleanup(db *gorm.DB) (bool, error) {
 	return false, nil
 }
 
-func (job *Job) GetEnqueJobs(resourceTypes []string, since string, retrieveNewBeneHistData bool) (enqueJobs []*que.Job, err error) {
```

Comment:
Functionality moved to [GetQueJobs](https://github.com/CMSgov/bcda-app/pull/568/files#diff-6ededccbab0684f8a6262e436323c065R83)